### PR TITLE
feat/cwp

### PR DESCRIPTION
**Pull request by SevenEarth**
feat/cwp

### CODE REVIEW

File: tencentcloud/resource_tc_cwp_license_order.go [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+			"alias": {
+				Optional:    true,
+				Computed:    true,
+				Type:        schema.TypeString,
+				Description: "Resource alias.",
+			},
+			"license_type": {
+				Optional:     true,
+				Type:         schema.TypeInt,
+				Default:      LICENSE_TYPE_0,
```
Comment (andrew-tx) [inline tencentcloud/resource_tc_cwp_license_order.go:61]: Default字段是否必须

File: tencentcloud/resource_tc_cwp_license_order.go [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+				ValidateFunc: validateAllowedIntValue(LICENSE_TYPE),
+				Description:  "LicenseType, 0 CWP Pro - Pay as you go, 1 CWP Pro - Monthly subscription, 2 CWP Ultimate - Monthly subscription. Default is 0.",
+			},
+			"license_num": {
+				Optional:    true,
+				Type:        schema.TypeInt,
+				Default:     1,
```
Comment (andrew-tx) [inline tencentcloud/resource_tc_cwp_license_order.go:68]: Default字段是否必须

File: tencentcloud/resource_tc_cwp_license_order.go (hunk 1 above)
Comment (SevenEarth) [reply tencentcloud/resource_tc_cwp_license_order.go:61]: SDK默认0

File: tencentcloud/resource_tc_cwp_license_order.go (hunk 2 above)
Comment (SevenEarth) [reply tencentcloud/resource_tc_cwp_license_order.go:68]: SDK默认1

**Review by andrew-tx — APPROVED** [review]
LGTM

**Pull request merged**